### PR TITLE
BaSP-TG- 61- Super Admins - Redux Implementation

### PR DESCRIPTION
**Pull request by EzeRocco**
- [x] Set actions and constants
- [x] Set reducer
- [x] Implement thunk
- [x] Implement List
- [x] Implement Delete
- [x] Implement Add
- [x] Implement Edit

### CODE REVIEW

File: src/redux/superAdmins/thunks.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  deleteSuperAdminPending,
+  deleteSuperAdminSuccess,
+  deleteSuperAdminError,
+  editSuperAdminPending,
+  editSuperAdminSuccess,
+  editSuperAdminError
+} from './actions';
+
+export const getSuperAdmins = () => {
+  return (dispatch) => {
```
Comment (JuanMornik) [inline src/redux/superAdmins/thunks.js:17]: Hace esta función asincrónica, asi te queda consistente con las otras

**Review by JuanMornik — CHANGES_REQUESTED** [review]

**Review by JuanMornik — APPROVED** [review]

**Review by NancyCima — APPROVED** [review]
Nice Job!!

Remember to check the tasks you completed on the PR's description 👍

**Review by CristianAlcober — CHANGES_REQUESTED** [review]
Hi Eze, I just saw everything, and I have to say that almost everything seems to work nice, there is just one issue:

- [ ] When adding or editing an item to the list, if you fill wrong one (or more) of the fields, or if you leave them empty and then click on the confirmation button, all the screen turns white and will not change unless you reload the page.

That's it, is just a minor thing to correct, however it is necessary. Everything else works splendidly!

**Review by JuanMornik — CHANGES_REQUESTED** [review]

File: src/Components/SuperAdmins/index.js [hunk 2]
```diff
@@ -22,90 +24,46 @@ const SuperAdmins = () => {
...
+          />
+          <Form
+            showForm={showForm}
+            setShowForm={setShowForm}
+            previousSuperAdmin={previousSuperAdmin}
+            setPreviousSuperAdmin={setPreviousSuperAdmin}
+            setShowModal={setShowModal}
+            setChildrenModal={setChildrenModal}
+          />
+          <Button onClick={onClick}> Add Super Admin</Button>
```
Comment (LucianoAlarcon) [inline src/Components/SuperAdmins/index.js:59]: Aca podrias borrarr la funcion de onClick ya que solamente le estas pasando el valor del setShowForm y ejecutar ese estado como funcion directamente
```suggestion
          <Button onClick={() => setShowForm(true)}> Add Super Admin</Button>
```

**Review by LucianoAlarcon — CHANGES_REQUESTED** [review]

**Review by CristianAlcober — APPROVED** [review]
Excellent job Eze, everything works nicely, thanks for fixing it!

**Review by LucianoAlarcon — APPROVED** [review]

**Pull request merged**